src/viewers/PictureInPictureViewer.js

```diff
@@ -63,6 +92,11 @@ const PictureInPictureViewer = props => {
     transparentColor,
     onViewStateChange
   } = props;
+  const {
+    newLoaderSelection,
+    oldLoaderSelection,
+    onViewportLoad
+  } = useGlobalSelection(loaderSelection);
```

Comment:
Any reason we can't also do this on the `SideBySideViewer`?

Reply:
No, it should work for `SideBySideViewer`, too. I can have a look later today.

Reply:
A more recent commit implements it for `SideBySideViewer`.